src/views/Statistic/StatisticGroup.js

```diff
@@ -0,0 +1,68 @@
+import _ from 'lodash'
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+
+import { customPropTypes, getUnhandledProps, useKeyOnly } from '../../utils/propUtils'
+import numberToWord from '../../utils/numberToWord'
+import META from '../../utils/Meta'
+import Statistic from './Statistic'
+
+function StatisticGroup(props) {
+  const { children, className, horizontal, items, widths } = props
+  const classes = cx(
+    'ui',
+    useKeyOnly(horizontal, 'horizontal'),
+    _.isNumber(widths) ? numberToWord(widths) : widths,
+    className,
+    'statistics'
+  )
+  const rest = getUnhandledProps(StatisticGroup, props)
+
+  if (!items) {
+    return <div className={classes} {...rest}>{children}</div>
+  }
+
+  let itemsJSX = []
+
+  items.forEach((item, index) => {
+    const key = `${item.label}-${item.value}-${index}`
+    itemsJSX.push(<Statistic key={key} {...item} />)
+  })
+
+  return <div className={classes} {...rest}>{itemsJSX}</div>
```

Comment:
Have to correct myself here, while this is true regarding prop order `rest` actually does not include className in this example.  That is because the className prop is being handled in the propTypes, so `getUnhandledProps` is returning an object of props that does not include `className`.

Let's still spread them first, and I'm adding the common test now. I just wanted to point out that this would actually work as is and my original comment was incorrect.